quickwit-cli/src/service.rs

```diff
@@ -70,15 +69,23 @@ impl RunCliCommand {
             .map(Uri::try_new)
             .expect("`config` is a required arg.")?;
         let data_dir_path = matches.value_of("data-dir").map(PathBuf::from);
-        let services: HashSet<QuickwitService> =
-            if let Some(service_str) = matches.value_of("service") {
-                let service = QuickwitService::try_from(service_str)?;
-                iter::once(service).collect()
-            } else {
-                [QuickwitService::Indexer, QuickwitService::Searcher]
-                    .into_iter()
-                    .collect()
-            };
+        let services = matches
+            .value_of("service")
+            .map(|services_str| {
+                let services: Result<HashSet<_>, _> = services_str
+                    .split(',')
```

Comment:
why not several flags?

Reply:
true, better to used `multiple_occurence` from clap.